src/terrain/terrain.cpp

```diff
@@ -25,9 +25,8 @@ namespace terrain {
 
 int Terrain::Area_size = 32;
 
-Terrain::Terrain() {
-    init_old(1, 1, 1);
-    seed = 0;
+Terrain::Terrain() : seed(0) {
+    init_old(0, 0, 0);
```

Comment:
I thought you got rid of the `init*` things

Reply:
This is quite complicated. If you want this changed you can make an issue, and assign it to me.

Reply:
Uhh just rename it to `init` and add a TODO